### PR TITLE
add code of conduct

### PR DESCRIPTION
**Pull request by tonymrakovcic**
Please comment if all of this makes sense, and what else we could add

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,13 +3,15 @@
 This project is work of [many contributors](https://github.com/bornfight/README/graphs/contributors).
```
Comment (napravicukod) [inline CONTRIBUTING.md]: Is there a link somewhere to this file, how can I access it from the initial readme?
Or is the idea to click on the actual file

File: getting-started/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Code of conduct for this repo
+description: Some simple rules to follow
+---
+
+Here is a list of things to keep in mind when contributing and participating in this repo:
+
+- Never place tokens, passwords or any other sensitive data into your commits
```
Comment (napravicukod) [inline getting-started/code-of-conduct.md]: Add punctuation at the end if these are sentences maybe.

File: CONTRIBUTING.md (hunk 1 above)
Comment (tonymrakovcic) [reply CONTRIBUTING.md]: this is githubs link :) https://github.com/bornfight/README/graphs/contributors
like /issues or /pulls 
It just shows a list of people that commit to the repo and fancy charts and graphs

File: getting-started/code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- Respect GDPR.
+- If you have copy pasted some code/text from somewhere please provide a link to the original.
+- Do not name clients and projects by name, you never know if they want to stay anonymous or their project is not.
+  for the public eye yet.
+- Don't forget repo links, although restricted, contain project names too, so do not include these also.
+- Please always open a pull request for new edits, and do not commit directly to the master branch.
+- Since this repo is intended to be public, please add a little 🔒 sign to any link that is closed so that it is.
+  easily visible that the content is not public.
+- If you are not sure if something is ok to put in this repo, you can open an issue, ask on slack.
+- Please be nice, in comments, PRs, repo content and in life!
```
Comment (jakagacic) [inline getting-started/code-of-conduct.md]: 👍

**Pull request merged**